y_lb.libraries.yml

```diff
@@ -2,16 +2,22 @@ layout_builder:
   version: 1.0
```

Comment:
If we do the change, we need to bump library version string, weird cache issues otherwise